exercises/concept/take-a-number/.docs/hints.md

```diff
@@ -8,16 +8,17 @@
 ## 1. Start the machine
 
 - The machine should run in a new process. There is [a built-in function that starts a new process][kernel-spawn-1].
-- You will need another function that the new process will execute. You can name it, for example, `loop`.
-- Use the [capture operator][special-forms-capture] to pass a named function as an argument.
+- Use a 0-arity anonymous function when starting a new process. This function doesn't need to do anything yet.
 
 ## 2. Report the machine state
 
-- The machine's process needs to respond to messages.
+- You will need a new named function. You can name it, for example, `loop`.
+- This new function should accept one argument, the state.
+- The 0-arity anonymous function from the previous step can call the new named function, passing the initial state.
```

Comment:
I hope those instructions are explicit enough to lead people to a solution similar to the exemplar

Reply:
I think they should be